addon/components/au-dropdown.js

```diff
@@ -14,13 +14,14 @@ export default class AuDropdown extends Component {
   // Open dropdown
   @action
   openDropdown() {
-    // Toggle dropdown view state
+    // Open dropdown view state
```

Comment:
Nitpick: I would simply delete these kind of comments since they don't add much value over the method name anyways 😄.